src/config/i18n.ts

```diff
@@ -0,0 +1,9 @@
+import Language from '../types/i18n/Language';
+
+export default {
+    language: Language.EN_GB,
+    languages: [
+        Language.EN_GB,
+        Language.NO,
+    ] as string[],
+};
```

Comment:
these are specific to us whereas I was trying to keep this generic so that it is appropriate for open-source